admin-front/src/components/Activities/DatetTimePicker.tsx

```diff
@@ -0,0 +1,52 @@
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
```

Comment:
There is a typo in the name of this .tsx file (Datet => Date)

Reply:
Renamed, thanks